tools/make_hgrid/make_hgrid.c

```diff
@@ -940,7 +1034,7 @@ int main(int argc, char* argv[])
 			 area, angle_dx, center, verbose, use_great_circle_algorithm);
   else if(my_grid_type==FROM_FILE) {
     for(n=0; n<ntiles; n++) {
-      long n1, n2, n3, n4;
+      int n1, n2, n3, n4;
```

Comment:
Why the change from `long` to `int`?

Reply:
Changed this back to the more-specific `long` type.